ci/pipelines/deploy-to-test.yml

```diff
@@ -1422,9 +1429,18 @@ jobs:
           AWS_ACCESS_KEY_ID: ((.:role.AWS_ACCESS_KEY_ID))
           AWS_SECRET_ACCESS_KEY: ((.:role.AWS_SECRET_ACCESS_KEY))
           AWS_SESSION_TOKEN: ((.:role.AWS_SESSION_TOKEN))
-      - put: adminusers-latest-ecr-registry-test
-        params:
-          image: adminusers-candidate-ecr-registry-test/image.tar
+      - in_parallel:
+        - do:
+          - put: adminusers-ecr-registry-test
+            params:
+              image: adminusers-candidate/image.tar
+              additional_tags: parse-candidate-tags/release-tag
+          - put: adminusers-latest
+            params:
+              image: adminusers-candidate/image.tar
+        - put: adminusers-dockerhub
```

Comment:
Am I right that, until we remove the Jenkins step, this `latest-master` image will already be in Dockerhub?

Reply:
That's correct, but it doesn't really matter, it _should_ be a quick push because it's using the same image so I think all these will say "existing blob"

Reply:
Yeah the one that ran:
<img width="1301" alt="Screenshot 2022-03-04 at 10 02 48" src="https://user-images.githubusercontent.com/2170030/156742595-99962c40-c133-40c3-bab2-1d8ff5746dcc.png">